spock-specs/src/test/groovy/org/spockframework/docs/primer/VerifyMethodsDocSpec.groovy

```diff
@@ -62,6 +62,41 @@ pc.clockRate >= 2333
 """ /* end::verify-helper-method-result[] */)
   }
 
+  def "simple static @Verify helper method"() {
+    when:
+    def result = runner.runSpecBody("""
+@groovy.transform.Canonical
+static class PC {
+    final String vendor
+    final int clockRate
+    final int ram
+    final String os
+}
+
+@Verify
+static matchesPreferredConfiguration(PC pc) {
+  pc.vendor == "Sunny"
+  pc.clockRate >= 2333
+  pc.ram >= 4096
+  pc.os == "Linux"
+}
```

Comment:
I don't see any tags for inclusion in the docs, or am I missing something?
I thought we agreed to show a short example for 

```
@Verify void method() { ... }
@Verify def method() { ... }
@Verify method() { ... }
```

Reply:
Are you [sure](https://groovyconsole.dev/?g=groovy_4_0&codez=eNpNi7EKwjAURfd8xaVT61DoKhTq4ubkF4Tk1QbTvJC8oiL-u2lwcLlwDue6NXIS5Mjm3nsdbv1BKeN1zjhdIxnQUyjYjB3c7IwWxwFvBVia0SzkPePBydum7apH-ZRYjqgADBhHrK8LycK27Yr9qDLTmflP_76JZEsBQ61KN7kglGZtCHu_R0V_AcFENmo)?